rclcpp/src/rclcpp/publisher_base.cpp

```diff
@@ -47,8 +47,24 @@ PublisherBase::PublisherBase(
 : rcl_node_handle_(node_base->get_shared_rcl_node_handle()),
   intra_process_is_enabled_(false), intra_process_publisher_id_(0)
 {
+  auto custom_deletor = [node_handle = this->rcl_node_handle_](rcl_publisher_t * rcl_pub)
```

Comment:
nit: deletor -> deleter ?

Reply:
> nit: deletor -> deleter ?

Heh.  I usually take the `-or` suffix as "the one who deletes".  But in this case, there doesn't seem to be a real word corresponding to this.  This is copied from https://github.com/ros2/rclcpp/blob/eff11d61bbb60220003779eb4a5a1b205a76da29/rclcpp/src/rclcpp/subscription_base.cpp#L47 , but I agree with you on `deleter`; I'll change it here and we can change it in the subscription_base some other time.

Reply:
I'm not a native English speaker, so you can ignore my comment :smile:.

I've seen `deleter` many times and I don't remember having seen `deletor`, so I thought it was a typo.